Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -910,6 +1014,133 @@ def delete_rpz_rule_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.ARecord(val.ReferenceID && val.ReferenceID === obj.ReferenceID)': context_data}
+    human_readable = tableToMarkdown(title, context_data, headerTransform=pascalToSpace)
+
+    return human_readable, context, raw_response
+
+
+def add_host_record_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    host = args.get('host')
+    ipadd = args.get('ipadd')
+    raw_response = client.add_host(host, ipadd)
+    ref_id = raw_response.get('result')
+    context_data = {RESPONSE_TRANSLATION_DICTIONARY['_ref']: ref_id}
+    title = f'{INTEGRATION_NAME} - Host Record {host} with IP address {ipadd} was added successfully with reference\n' \
+            f'{ref_id}'
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.Host(val.ReferenceID && val.ReferenceID === obj.ReferenceID)': context_data}
+    human_readable = tableToMarkdown(title, context_data, headerTransform=pascalToSpace)
+    return human_readable, context, raw_response
+
+
+def update_host_ip_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    ref_id = str(args.get("ref_id"))
+    ipv4addr = str(args.get("ipv4addr"))
+    raw_response = client.update_host_ip(ref_id, ipv4addr)
+    results = raw_response.get('result')
+    context_data = results_to_context_data(results, to_list=False)
+    title = f"{INTEGRATION_NAME} - Updated a Host's IP with new IP address: {ipv4addr}"
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.Host(val.ReferenceID && val.ReferenceID === obj.ReferenceID)': context_data}
+    human_readable = tableToMarkdown(title, context_data, headerTransform=pascalToSpace)
+    return human_readable, context, raw_response
+
+
+def update_a_record_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    ref_id = str(args.get("ref_id"))
+    ipv4addr = str(args.get("ipv4addr"))
+    name = str(args.get("name"))
+    comment = str(args.get("comment"))
+
+    raw_response = client.update_a_record(ref_id, ipv4addr, name, comment)
+    result_ref_id = raw_response.get('result')
+    context_data = {RESPONSE_TRANSLATION_DICTIONARY['_ref']: result_ref_id}
+    title = f'{INTEGRATION_NAME} - Updated a Host Record with the ReferenceID: {ref_id} with IP address: {ipv4addr}'
+    context = {
+        f'{INTEGRATION_CONTEXT_NAME}.ARecord(val.ReferenceID && val.ReferenceID === obj.ReferenceID)': context_data}
+    human_readable = tableToMarkdown(title, context_data, headerTransform=pascalToSpace)
+    return human_readable, context, raw_response
+
+
+def delete_host_record_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """
+    Args:
+        client: Client object
+        args: Usually demisto.args()
+
+    Returns:
+        Outputs
+    """
+
+    ref_id = args.get('ref_id')
+
+    if "/" in str(ref_id):
+        refIDStr = str(ref_id).split("/")
+        refIDStr = refIDStr[1].split(":")
+        ref_id = refIDStr[0]
```

Comment:
Why do you need this code?
are you trying to get the text wrapped by `/` and `:`? use regex: `/(.+):`